.github/CODEOWNERS

```diff
@@ -56,6 +56,8 @@
 # PRLabel: %Event Hubs
 /sdk/eventhub/                                                       @annatisch @kashifkhan @swathipil @l0lawrence
 
+# ServiceOwners: @jalauzon-msft @vincenttran-msft
+# ServiceLabel: %Storage
 # PRLabel: %Storage
 /sdk/storage/                                                        @annatisch @jalauzon-msft @vincenttran-msft
```

Comment:
```suggestion
# ServiceOwners: @jalauzon-msft @vincenttran-msft
# ServiceLabel: %Storage

# PRLabel: %Storage
/sdk/storage/                                                        @annatisch @jalauzon-msft @vincenttran-msft
```

If we don't split the blocks here, Anna will be considered part of the service team and will get pulled in for all "Service Attention" labels.